### PR TITLE
Ignore error on ServiceClass already exists

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: pkg/primaza/controlplane/serviceclasses.go [hunk 1]
```diff
@@ -35,7 +36,9 @@ func PushServiceClassToNamespaces(ctx context.Context, cli client.Client, sc pri
 		}
 
 		if err := cli.Create(ctx, sccp, &client.CreateOptions{}); err != nil {
-			return err
+			if !apierrors.IsAlreadyExists(err) {
+				return err
+			}
```
Comment (sadlerap) [inline pkg/primaza/controlplane/serviceclasses.go:41]: If it already exists, don't we want to update it?

Comment (filariow) [reply pkg/primaza/controlplane/serviceclasses.go:41]: yes, eventually this will be removed in favor of update logic. For now, it clears a bit the logs

Comment (filariow) [reply pkg/primaza/controlplane/serviceclasses.go:42]: done

**Comment by filariow** [comment]
superseded by #135

**Pull request closed without merging**